packages/xstate-immer/src/index.ts

```diff
@@ -5,7 +5,9 @@ import {
   assign as xstateAssign,
   AssignMeta
 } from 'xstate';
-import { produce, Draft } from 'immer';
+import { produce, Draft, Patch, enablePatches, applyPatches } from 'immer';
+
+enablePatches();
```

Comment:
I'm not super comfortable with a library calling something like that - it is a side effect after all. I understand why this is needed - but maybe this call should be required from a user? Is there any guidance from Immer on this for libraries integrating with Immer? cc @mweststrate

Reply:
If your library is using patches, this method should indeed be called. It prevents the patch functionality from being tree-shaken out of the final bundle, which is what you want :) (you could make that conditional in your library as well if patches are not always used). It saves < 1kb gzipped, so it might not be worth the additional complexity, but was introduced to make sure that immer can be used as minimally as possible. 

Enabling this doesn't semantically affect any other consumers of Immer, and calling it multiple times is safe. So doing this here is safe and probably the correct thing to do.